client/src/views/About.vue

```diff
@@ -21,5 +21,8 @@
         <a v-t="'about.coffee'" href="https://www.buymeacoffee.com/ConorMurphy/" />
       </template>
     </i18n-t>
+    <router-link to="/">
+      <button v-t="'homeButton'" type="button" class="btn btn-orange w-10" />
```

Comment:
all buttons should have a sound played when pressed, you can look at other buttons for examples on how to do this

Reply:
I used the default click1.mp3, let me know if there is a different sound that should be used.